version: '3.9'

services:
  dash:
    command: python3 app.py
    image: anubhavkharel01/a3:latest
    environment:
      - HOST=0.0.0.0
      - PORT=80
      - MLFLOW_TRACKING_URI=https://mlflow.ml.brain.cs.ait.ac.th
      - MLFLOW_TRACKING_USERNAME=admin
      - MLFLOW_TRACKING_PASSWORD=password
    labels:
      - traefik.enable=true
      - traefik.http.services.dash-st125999.loadbalancer.server.port=80
      - traefik.http.routers.dash-st125999.rule=Host(`st125999.ml.brain.cs.ait.ac.th`)
      - traefik.http.routers.dash-st125999.tls=true
      - traefik.http.routers.dash-st125999.tls.certresolver=production
      - traefik.http.routers.dash-st125999.entrypoints=websecure
    networks:
      - default
      - traefik_default

networks:
  default:
  traefik_default:
    external: true
